CSS folder/index.css

```diff
@@ -0,0 +1,269 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Times New Roman", Times, serif;
+  line-height: 1.4;
+}
+
+a {
+  text-decoration: none;
+}
+
+p {
+  margin: 0.75rem 0;
+}
+.text-primary {
+  color: #ffa500;
+}
+
+.navbar {
+  display: flex;
+  background: #333;
+  color: #fff;
+  position: sticky;
+  top: 0;
+  justify-content: space-between;
+  z-index: 1;
+  padding: 16px;
+}
+.navbar ul {
+  display: flex;
+  align-items: center;
+  list-style: none;
+}
+
+.navbar ul li a {
+  color: #fff;
+  padding: 0.75rem;
+  margin: 0 0.25rem;
+}
+
+.navbar ul li a:hover {
+  background: #ffa500;
+  border-radius: 5px;
+}
+
+/* Header*/
+.headers {
+  background: #333 url("../images/office-computer.jpg") no-repeat center
+    center/cover;
+  height: 100vh;
+  color: #fff;
+}
+
+.headers div {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  padding: 0 2rem;
+  /* overlay */
+  position: absolute;
+  top: 62px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(0, 0, 0, 0.4);
+}
+
+.l-heading {
+  font-size: 4rem;
+  margin-bottom: 0.75rem;
+  line-height: 1.1;
+}
+
+.m-heading {
+  font-size: 2rem;
+  margin-bottom: 0.75;
+  line-height: 1.1;
+}
+
+.lead {
+  font-size: 1.3rem;
+  margin-bottom: 2rem;
+}
+.sub {
+  text-align: center;
+  margin-top: 1px;
+}
+.py-1 {
```

Comment:
You've done a good job of naming your classes. This one isn't one of them. Please be a little more descriptive with your naming